webserver/webserver/src/main/java/io/helidon/webserver/StaticContentSupport.java

```diff
@@ -195,6 +196,17 @@ public Builder welcomeFileName(String welcomeFileName) {
             return this;
         }
 
+        /**
+         * Sets custom temporary folder for extracting static content from a jar.
+         *
+         * @param tempFolder custom temporary folder
+         * @return updated builder
+         */
+        public Builder tempFolder(Path tempFolder) {
```

Comment:
please rename to `temporaryDirectory` or `tmpDir`. I think both `temp` and `folder` are less used

Reply:
np